### DISCUSSION TITLE
Model works in WebXR, but keeps crashing in Scene Viewer

### DISCUSSION THREAD

**Discussion by NadimSKanaan** [google/model-viewer#1807] [Q&A] [answered]
Hello,

I am using Model Viewer to Launch Scene Viewer for a web-based AR experience. I validated my 3D model on https://arvr.google.com/scene-viewer-preview and I had one error: The provided user input has a binary size that exceeds the maximum limit supported by the Scene Viewer specification, which is a maximum size limit of 15 MB. The recommended size limit is 10 MB. I had a look at the model used on UNICEF's website https://www.unicef.org/innovation/stories/girls-satellites-soar-stars and it turns out it also exceeded the 15MB limit. But, UNICEF's model works and mine doesn't. I have no other errors whatsoever and the model works fine with WebXR. Can someone let me know what could possibly be going wrong?
MY MODEL: https://cdn.glitch.com/2ff2a36b-6d14-4205-b7ab-8b050166c154%2FcamelHK_walk_particle_river.glb?v=1610384410708

**Comment by elalish** [comment, marked as answer]
@tpsiaki for SceneViewer info. In general I would say you should put a lot of effort into reducing model size, as that will relieve pressure on performance, GPU RAM, etc, which are complicated enough that it is hard to represent exact bounds for a file. We want to have more tools to help with this in our Editor eventually.
